pages/science/reactions.rst

```diff
@@ -70,30 +76,9 @@ where :math:`C_{\mathrm{k}}` is the concentration of species :math:`\mathrm{k}`.
 collision efficiency can be absorbed into the rate coefficient :math:`k_f(T)`, the default collision
 efficiency is 1.0.
 
-A three-body reaction may be defined using the :cti:class:`three_body_reaction`
```

Comment:
If it's not part of the diff, then that's what was here before. You're welcome to make a commit implementing this change.

Reply:
I know, I thought it would be easy enough for you to make the change here since you have the file already open to make other changes and it is related to some of the other areas that have changed.